### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by aPlacuzzi**
* Update `git clone` instruction with submodules
* Add fork instruction to specify that a secret with name `AUTOMERGE_TOKEN` is required by `automerge` action

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,6 +165,13 @@ Contributions to this project are welcome. Just some rules:
 * Stay in sync with the `develop` branch: pull often from `develop` (if the build passes), so that you don't diverge too much from the main development line.
 * Do not introduce low quality code. All the new code must comply with the checker rules (that are quite strict) and must not introduce any other warning. Resolutions of existing warnings (if any is present) are very welcome instead.
 
+#### Forking the project
+This project defines several CI jobs. One of them (`automerge`) requires a secret with:
+
+* name = AUTOMERGE_TOKEN
+* value = a github token with `public_repo` scope
```
Comment (DanySK) [inline README.md:172]: Add a link for the token generation, should be portable.

File: README.md [hunk 2]
```diff
@@ -165,6 +165,13 @@ Contributions to this project are welcome. Just some rules:
 * Stay in sync with the `develop` branch: pull often from `develop` (if the build passes), so that you don't diverge too much from the main development line.
 * Do not introduce low quality code. All the new code must comply with the checker rules (that are quite strict) and must not introduce any other warning. Resolutions of existing warnings (if any is present) are very welcome instead.
 
+#### Forking the project
```
Comment (DanySK) [inline README.md:168]: Move this before "Importing" and explain why one should fork instead of cloning directly

File: README.md [hunk 3]
```diff
@@ -165,6 +165,13 @@ Contributions to this project are welcome. Just some rules:
...
+This project defines several CI jobs. One of them (`automerge`) requires a secret with:
+
+* name = AUTOMERGE_TOKEN
+* value = a github token with `public_repo` scope
+
+If you don't provide it, then the CI workflow will fail.
```
Comment (DanySK) [inline README.md:174]: ...even though the project status is correct.
So I'd say that the Workflow may appear as failed even though the build process is working correctly

**Review by DanySK — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aPlacuzzi) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (aPlacuzzi) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (aPlacuzzi) [reply README.md]: Done.

**Pull request merged**